### PR TITLE
update: base model and settings_dev

### PR DESCRIPTION
**Pull request by ChogirmaliYigit**

### CODE REVIEW

**Review by ChogirmaliYigit — COMMENTED** [review]
Good

**Review by Chogirmali — APPROVED** [review]
Done

File: backend/apps/core/models.py [hunk 1]
```diff
@@ -3,7 +3,21 @@
 from django.db import models
 
 
-class S3Attachment(models.Model):
+class BaseModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
```
Comment (Chogirmali) [inline backend/apps/core/models.py:7]: good

**Review by Chogirmali — APPROVED** [review]

**Review by Chogirmali — CHANGES_REQUESTED** [review]
done

**Review by Chogirmali — APPROVED** [review]

**Pull request merged**